source/includes/_invoice-payment.md

```diff
@@ -1042,9 +751,6 @@ class CustomField {
    }
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?